### DISCUSSION TITLE
Shared vaults, folders, or files

### DISCUSSION THREAD

**Discussion by drmartell** [vrtmrz/obsidian-livesync#263] [Ideas]
I have several family members with whom I'd like to share information in Obsidian.

Some things that would be cool:
- shared vaults like [Obsidian sync](https://obsidian.md/sync)
- shared folders, like a lightweight version of how [LastPass](https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/c_about_shared_folders.html&_LANG=enus) implements them
- shared files, something like LastPass [shared items](https://support.lastpass.com/s/document-item?language=en_US&bundleId=lastpass&topicId=LastPass/t_sharing_share_item.html&_LANG=enus)

Anyhow, has anyone given thought as to how the livesync infrastructure could be extended to support one or more of these sharing strategies?

**Comment by drmartell** [comment]
Just did some further digging in the discussions here and saw this: https://github.com/vrtmrz/obsidian-livesync/discussions/152
So maybe some of this is already possible with [LiveSync Classroom](https://github.com/vrtmrz/livesync-classroom)?

**Reply by vrtmrz** [reply]
Sorry for being late! 
LiveSync-Classroom or FileSystem-LiveSync would be useful if we can share files within the whole of some specific folder.
Both of these replicators decrypt files from the source and store them in the destination with its encryption. Only the replicator knows each passphrase. Therefore, we do not have to care about the passphrases which we should not know.
However, if we want to share the file independently as mentioned original case, it would require some additional work.

I hope that this would answer your question!
